scripts/easy_train.py

```diff
@@ -0,0 +1,1844 @@
+    parser.add_argument("--epoch-size", type=int, default=100000000, dest='epoch_size', help="Number of positions per epoch.")
+    parser.add_argument("--validation-size", type=int, default=1000000, dest='validation_size', help="Number of positions per validation step.")
+    parser.add_argument("--tui", type=str2bool, default=True, dest='tui', help="Whether to show a nice TUI.")
+    parser.add_argument("--do-network-testing", type=str2bool, default=True, dest='do_network_testing', help="Whether to test networks as they are generated.")
+    parser.add_argument("--do-network-training", type=str2bool, default=True, dest='do_network_training', help="Whether to train networks.")
+    parser.add_argument("--network-testing-threads", type=int, default=default_testing_threads, dest='network_testing_threads', help="Number of threads to use for network testing. By default the available number of threads - minus data loader and pytorch threads. The optimal value might depend on the --threads, --num-workers and other machine load.")
+    parser.add_argument("--network-testing-explore-factor", type=float, default=1.5, dest='network_testing_explore_factor', help="Elo error estimates are multiplied by this amount to determine testing candidates.")
+    parser.add_argument("--network-testing-book", type=str, default='https://github.com/official-stockfish/books/raw/master/UHO_XXL_+0.90_+1.19.epd.zip', dest='network_testing_book', help="Path to a suitable book, or suitable link see https://github.com/official-stockfish/books")
+    parser.add_argument("--network-testing-time-per-game", type=float, default=None, dest='network_testing_time_per_game', help="Number of seconds per game")
+    parser.add_argument("--network-testing-time-increment-per-move", type=float, default=None, dest='network_testing_time_increment_per_move', help="Number of seconds added to clock per move")
+    parser.add_argument("--network-testing-nodes-per-move", type=int, default=None, dest='network_testing_nodes_per_move', help="Number of nodes per move to use for testing. Overrides time control. Should be used ove time control for better consistency.")
+    parser.add_argument("--network-testing-hash-mb", type=int, default=8, dest='network_testing_hash_mb', help="Number of MB of memory to use for hash allocation for each engine being tested.")
+    parser.add_argument("--network-testing-games-per-round", type=int, default=20 * default_testing_threads, dest='network_testing_games_per_round', help="Number of games per round to use. Essentially a testing batch size. By default uses larger batches with larger --network-testing-threads")
+    parser.add_argument("--resume-training", type=str2bool, default=False, dest='resume_training', help="Attempts to resume each run from its latest checkpoint.")
```

Comment:
I wonder if we should make this resume-training True by default, really useful from my point of view (easily quit training and resume, a real life-saver).